src/neomaril_codex/__utils.py

```diff
@@ -70,3 +71,15 @@ def refresh_token(login: str, password: str, base_url: str):
         return respose.json()["Token"]
     else:
         raise AuthenticationError(respose.text)
+
+
+def parse_json_to_yaml(json_dict) -> str:
+    """Parse a json file to yaml format
```

Comment:
Does this function work with a json on memory too?

Reply:
If you try running the following code:

```python
import yaml

d = yaml.load("/path/to/file.json", Loader=yaml.CLoader)
print(d)
o = yaml.dump(d)
print(o)
```

it won’t display anything meaningful. This is because the file is in JSON format, not YAML. To fix this, you should load the file using the json module instead:

```python
import yaml
import json

with open("/home/henrique/datarisk/neomaril/samples/autoML/conf.json") as f:
    d = json.load(f)
print(d)
o = yaml.dump(d)
print(o)
```

Reply:
Maybe change the name of the function to `parse_dict_to_yaml`

Reply:
I meant the documentation, it states that this function works with a json file instead of a json in memory.

Reply:
https://github.com/datarisk-io/mlops-neomaril-codex/pull/59/commits/3e5754974e16607c13f3f2765babda81d6c064db